repositories/scenario_testrun.go

```diff
@@ -90,6 +116,21 @@ func (repo *MarbleDbRepository) GetActiveTestRunByScenarioIterationID(ctx contex
 	)
 }
 
+func (repo *MarbleDbRepository) GetTestRunByLiveVersionID(ctx context.Context,
```

Comment:
there can be more than one, so we need to be more specific here - I haven't finished reading, but probably the query should order by createdAt desc and take the most recent one / apply logic ?